### ISSUE TITLE
undefiend output

### ISSUE THREAD

**Issue by mjalili27** [talkhabi/vue-persian-datetime-picker#191]
![pdate](https://user-images.githubusercontent.com/31030497/141084101-a6e3ec04-4f8d-49de-8efe-24dde176a4d6.png)
hi, i am going to use this datepicker in [nextcloud server](https://github.com/nextcloud/server) but the output we get is like the attachment. how can we fix it ?
tnx

**Comment by mohsensaeedi** [comment]
Hi Mr @talkhabi 
First of all, I would like to thank you for your efforts in producing this package. but we have a similar problem that other people have reported here in 3 issues which probably should be merged. 

We also tested some previous versions without success. We used your recipe and tried different ways to solve the problem. Do you think this is definitely a bug? Do you think this issue will be resolved soon?

Thank you for your support

**Comment by talkhabi** [comment]
Hi Mr @mohsensaeedi thank you.
Yes this is a problem with build module (I guess `vue-cli`), and I'm working on it.
Currently can be fix by importing like this:
```js
import VuePersianDatetimePicker from 'vue-persian-datetime-picker/src/VuePersianDatetimePicker.vue'
```

**Comment by talkhabi** [comment]
fixed in `2.10.1`

**Comment by mohsensaeedi** [comment]
Mr @talkhabi 
Everything is working like charm. thank you man! 
خدا قوت :)
